build.py

```diff
@@ -225,7 +232,10 @@ def compile_js(out_path, js_files, level, externs):
 
   print('Writing', out_path)
   os.makedirs(os.path.dirname(out_path), exist_ok=True)
-  open(out_path, 'w').write(result['compiledCode'])
+  if result.get('compiledCode'):
+    open(out_path, 'w').write(result.get('compiledCode'))
+  else:
+    print(missing_key_string('compiledCode'))
```

Comment:
How odd! In what case do you miss this key in the response?

Reply:
This case would actually make more sense to pring a more descriptive error message saying that compilation failed. I might write another patch to check for server errors in the closure response, as they might expain the reason why this is sometimes missing?